backend/apps/matching/templates/staff/staff_profile.html

```diff
@@ -0,0 +1,81 @@
+   {% for message in messages %}
+  <div class="alert {{ message.tags }} alert-dismissible" role="alert">
+    <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+      <span aria-hidden="true">&times;</span>
+    </button>
+    {{ message }}
+  </div>
+{% endfor %}
+
+    <p>
+        <a role="button" class="btn blue text-white btn-md" href="logout">{% blocktrans %}Log out{% endblocktrans %}</a>
+        <a role="button" class="btn blue text-white btn-md" href="/matching/password_change">{% blocktrans %}Change Password{% endblocktrans %}</a>
+    </p>
+    <hr>
+    <h3>{% blocktrans %}Functions for staff members{% endblocktrans %}</h3>
+    <hr>
+    <p>
+        <a role="button" class="btn blue text-white btn-md" href="/matching/profile_staff">
+        <svg class="bi bi-check-all" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" d="M12.354 3.646a.5.5 0 010 .708l-7 7a.5.5 0 01-.708 0l-3.5-3.5a.5.5 0 11.708-.708L5 10.293l6.646-6.647a.5.5 0 01.708 0z" clip-rule="evenodd"/>
+          <path d="M6.25 8.043l-.896-.897a.5.5 0 10-.708.708l.897.896.707-.707zm1 2.414l.896.897a.5.5 0 00.708 0l7-7a.5.5 0 00-.708-.708L8.5 10.293l-.543-.543-.707.707z"/>
+        </svg>
+        {% blocktrans %}Do something{% endblocktrans %}</a>
+    </p>
+    <p>
+        <a role="button" class="btn blue text-white btn-md" href="/matching/profile_staff">
+        <svg class="bi bi-newspaper" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor"
+             xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd"
+                  d="M0 2A1.5 1.5 0 011.5.5h11A1.5 1.5 0 0114 2v12a1.5 1.5 0 01-1.5 1.5h-11A1.5 1.5 0 010 14V2zm1.5-.5A.5.5 0 001 2v12a.5.5 0 00.5.5h11a.5.5 0 00.5-.5V2a.5.5 0 00-.5-.5h-11z"
+                  clip-rule="evenodd"/>
+            <path fill-rule="evenodd"
+                  d="M15.5 3a.5.5 0 01.5.5V14a1.5 1.5 0 01-1.5 1.5h-3v-1h3a.5.5 0 00.5-.5V3.5a.5.5 0 01.5-.5z"
+                  clip-rule="evenodd"/>
+            <path d="M2 3h10v2H2V3zm0 3h4v3H2V6zm0 4h4v1H2v-1zm0 2h4v1H2v-1zm5-6h2v1H7V6zm3 0h2v1h-2V6zM7 8h2v1H7V8zm3 0h2v1h-2V8zm-3 2h2v1H7v-1zm3 0h2v1h-2v-1zm-3 2h2v1H7v-1zm3 0h2v1h-2v-1z"/>
+        </svg>
+        {% blocktrans %}Do something{% endblocktrans %}</a>
+    </p>
+    <p>
+        <a role="button" class="btn blue text-white btn-md" href="/use_statistics/view">
+        <svg class="bi bi-bar-chart" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" d="M4 11H2v3h2v-3zm5-4H7v7h2V7zm5-5h-2v12h2V2zm-2-1a1 1 0 00-1 1v12a1 1 0 001 1h2a1 1 0 001-1V2a1 1 0 00-1-1h-2zM6 7a1 1 0 011-1h2a1 1 0 011 1v7a1 1 0 01-1 1H7a1 1 0 01-1-1V7zm-5 4a1 1 0 011-1h2a1 1 0 011 1v3a1 1 0 01-1 1H2a1 1 0 01-1-1v-3z" clip-rule="evenodd"/>
+        </svg>
+        {% blocktrans %}Do something{% endblocktrans %}</a>
```

Comment:
Shouldn't this already be adjusted?
```suggestion
        {% blocktrans %}Show usage stats{% endblocktrans %}</a>
```

Reply:
At some point somebody will have to make the page pretty anyway and set the right links :)

Reply:
Using placeholders where there is no function is probably the clearer version but having the possibility to view the statistics would be good